Lib/test/test_pprint.py

```diff
@@ -187,6 +208,21 @@ def test_same_as_repr(self):
                              .replace('\n', ' '), native)
             self.assertEqual(pprint.saferepr(simple), native)
 
+    def test_repr_override_called(self):
+        N = 1000
+        # Ensure that a __repr__ override is called for subclasses of containers.
+        for simple in (list_custom_repr([]), list_custom_repr([1,2,3]), list_custom_repr(range(N)),
```

Comment:
These lines seem too long. PEP 8 sets limit of line length for code to 79 characters.

Reply:
Thanks, will fix.